WebGL: Fix description of vertexAttrib (#29772)
mdn/content: files/en-us/web/api/webglrenderingcontext/vertexattrib/index.md

Attributes may be matrices, in which case columns of the matrix must be loaded into
successive vertex attribute slots.

> The values set with `vertexAttrib` are context-global, i.e. they aren't part of the shader state

Reviewer: ```suggestion
The values set with `vertexAttrib` are context-global; this is, they aren't part of the shader state
```

> The values set with `vertexAttrib` are context-global; this is, they aren't part of the shader state

Reviewer: ```suggestion
The values set with `vertexAttrib` are context-global; that is, they aren't part of the shader state
```
